2013/06/09 22:19:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "1".' in Z:\home\localhost\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/1
---
2013/06/11 21:49:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "my".' in Z:\home\localhost\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my')
#1 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/my
---
2013/06/11 21:50:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "my".' in Z:\home\localhost\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my')
#1 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/my
---
2013/06/11 21:54:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "my/index".' in Z:\home\localhost\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#1 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/11 21:54:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "my/index".' in Z:\home\localhost\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#1 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/11 21:54:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "my/index".' in Z:\home\localhost\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#1 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/11 21:54:59 [error] [exception.CException] exception 'CException' with message 'Контроллер MyController не может найти представление "index".' in Z:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(12): CController->render('index')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/11 21:55:55 [error] [exception.CException] exception 'CException' with message 'Контроллер MyController не может найти представление "index".' in Z:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(12): CController->render('index')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionOne()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('one')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/one')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/one
---
2013/06/11 21:56:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "one".' in Z:\home\localhost\www\yii\framework\web\CController.php:483
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(270): CController->missingAction('one')
#1 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('one')
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/one')
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/my/one
---
2013/06/11 22:00:46 [error] [exception.CException] exception 'CException' with message 'Контроллер MyController не может найти представление "my".' in Z:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('my', NULL, true)
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(12): CController->render('my')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/12 19:50:26 [error] [php] Cannot use a scalar value as an array (Z:\home\localhost\www\yii\protected\controllers\MyController.php:22)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (22)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 19:52:02 [error] [php] Cannot use a scalar value as an array (Z:\home\localhost\www\yii\protected\controllers\MyController.php:25)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (25)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 19:52:14 [error] [php] Cannot use a scalar value as an array (Z:\home\localhost\www\yii\protected\controllers\MyController.php:24)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (24)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 19:52:44 [error] [php] Cannot use a scalar value as an array (Z:\home\localhost\www\yii\protected\controllers\MyController.php:25)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (25)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 22:23:26 [error] [php] file() expects parameter 2 to be long, string given (Z:\home\localhost\www\yii\protected\controllers\MyController.php:29)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#4 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (29)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 22:23:49 [error] [php] file() expects parameter 2 to be long, string given (Z:\home\localhost\www\yii\protected\controllers\MyController.php:29)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#4 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (29)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 22:23:49 [error] [php] file() expects parameter 2 to be long, string given (Z:\home\localhost\www\yii\protected\controllers\MyController.php:29)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#4 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (29)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/12 22:24:23 [error] [php] file() expects parameter 2 to be long, string given (Z:\home\localhost\www\yii\protected\controllers\MyController.php:29)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#4 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (29)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/15 20:21:01 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Book._attributes:CActiveRecord".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_attributes:CAc...')
#1 Z:\home\localhost\www\yii\framework\base\CModel.php(595): CActiveRecord->__get('_attributes:CAc...')
#2 Z:\home\localhost\www\yii\protected\controllers\BookController.php(18): CModel->offsetGet('_attributes:CAc...')
#3 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): BookController->actionIndex()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('book/index')
#9 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/index.php/book/index
---
2013/06/15 20:21:12 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Book._attributes".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_attributes')
#1 Z:\home\localhost\www\yii\framework\base\CModel.php(595): CActiveRecord->__get('_attributes')
#2 Z:\home\localhost\www\yii\protected\controllers\BookController.php(18): CModel->offsetGet('_attributes')
#3 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): BookController->actionIndex()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('book/index')
#9 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/index.php/book/index
---
2013/06/15 20:21:12 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Book._attributes".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_attributes')
#1 Z:\home\localhost\www\yii\framework\base\CModel.php(595): CActiveRecord->__get('_attributes')
#2 Z:\home\localhost\www\yii\protected\controllers\BookController.php(18): CModel->offsetGet('_attributes')
#3 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): BookController->actionIndex()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('book/index')
#9 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/index.php/book/index
---
2013/06/15 20:21:13 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Book._attributes".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_attributes')
#1 Z:\home\localhost\www\yii\framework\base\CModel.php(595): CActiveRecord->__get('_attributes')
#2 Z:\home\localhost\www\yii\protected\controllers\BookController.php(18): CModel->offsetGet('_attributes')
#3 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): BookController->actionIndex()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('book/index')
#9 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/index.php/book/index
---
2013/06/15 20:45:50 [error] [php] Invalid argument supplied for foreach() (Z:\home\localhost\www\yii\protected\controllers\BookController.php:19)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(286): BookController->runAction()
#1 Z:\home\localhost\www\yii\framework\web\CController.php(265): BookController->runActionWithFilters()
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): BookController->run()
#3 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/book/index
in Z:\home\localhost\www\yii\protected\controllers\BookController.php (19)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/15 20:55:04 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchColumn(): SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SELECT COUNT(*) FROM i_book' at line 1. Выполнявшийся SQL-запрос: SELECT COUNT(*) FROM `i_book` `t` WHERE SELECT COUNT(*) FROM i_book.
in Z:\home\localhost\www\yii\protected\controllers\BookController.php (16)
in Z:\home\localhost\www\yii\index.php (13)
2013/06/15 20:55:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SELECT COUNT(*) FROM i_book' at line 1. The SQL statement executed was: SELECT COUNT(*) FROM `i_book` `t` WHERE SELECT COUNT(*) FROM i_book' in Z:\home\localhost\www\yii\framework\db\CDbCommand.php:541
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbCommand.php(432): CDbCommand->queryInternal('fetchColumn', 0, Array)
#1 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(1546): CDbCommand->queryScalar()
#2 Z:\home\localhost\www\yii\protected\controllers\BookController.php(16): CActiveRecord->count('SELECT COUNT(*)...')
#3 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): BookController->actionIndex()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('book/index')
#9 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/index.php/book/index
---
2013/06/16 01:23:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "_form".' in Z:\home\localhost\www\yii\framework\web\CController.php:483
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(270): CController->missingAction('_form')
#1 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('_form')
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('page/_form')
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/page/_form
---
2013/06/16 17:19:28 [error] [exception.CException] exception 'CException' with message 'В классе Page и его поведениях не найден метод или замыкание с именем "valid".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('valid', Array)
#1 Z:\home\localhost\www\yii\framework\validators\CInlineValidator.php(42): CActiveRecord->__call('valid', Array)
#2 Z:\home\localhost\www\yii\framework\validators\CInlineValidator.php(42): Page->valid('title', Array)
#3 Z:\home\localhost\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(Page), 'title')
#4 Z:\home\localhost\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(Page), NULL)
#5 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(794): CModel->validate(NULL)
#6 Z:\home\localhost\www\yii\protected\controllers\PageController.php(73): CActiveRecord->save()
#7 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): PageController->actionCreate()
#8 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 Z:\home\localhost\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 Z:\home\localhost\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 Z:\home\localhost\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 Z:\home\localhost\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 Z:\home\localhost\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 Z:\home\localhost\www\yii\framework\web\CController.php(291): CFilterChain->run()
#15 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('create')
#17 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('page/create')
#18 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/yii/index.php/page/create
HTTP_REFERER=http://localhost/yii/index.php/page/create
---
2013/06/16 20:46:45 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "My.text".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('text')
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(18): CActiveRecord->__get('text')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/16 20:46:45 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "My.text".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('text')
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(18): CActiveRecord->__get('text')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/16 20:47:56 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "My.text".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('text')
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(16): CActiveRecord->__get('text')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/16 20:47:56 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "My.text".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('text')
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(16): CActiveRecord->__get('text')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/16 20:47:57 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "My.text".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('text')
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(16): CActiveRecord->__get('text')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/16 20:48:03 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "My.text".' in Z:\home\localhost\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('text')
#1 Z:\home\localhost\www\yii\protected\controllers\MyController.php(19): CActiveRecord->__get('text')
#2 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#8 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/17 00:14:42 [error] [exception.CException] exception 'CException' with message 'Контроллер MyController не может найти представление "Z:\home\localhost\www\yii\protected\views\my/_page/_form".' in Z:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 Z:\home\localhost\www\yii\protected\views\my\index.php(12): CController->renderPartial('Z:\home\localho...', Array)
#1 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require('Z:\home\localho...')
#2 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('Z:\home\localho...', Array, true)
#3 Z:\home\localhost\www\yii\framework\web\CController.php(869): CBaseController->renderFile('Z:\home\localho...', Array, true)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#5 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): CController->render('index', Array)
#6 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#7 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#11 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#12 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/17 00:14:56 [error] [exception.CException] exception 'CException' with message 'Контроллер MyController не может найти представление "Z:\home\localhost\www\yii\protected\views\my\_page\_form".' in Z:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 Z:\home\localhost\www\yii\protected\views\my\index.php(12): CController->renderPartial('Z:\home\localho...', Array)
#1 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require('Z:\home\localho...')
#2 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('Z:\home\localho...', Array, true)
#3 Z:\home\localhost\www\yii\framework\web\CController.php(869): CBaseController->renderFile('Z:\home\localho...', Array, true)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#5 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): CController->render('index', Array)
#6 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#7 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#11 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#12 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/17 00:15:28 [error] [exception.CException] exception 'CException' with message 'Контроллер MyController не может найти представление "_form".' in Z:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 Z:\home\localhost\www\yii\protected\views\my\index.php(12): CController->renderPartial('_form', Array)
#1 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require('Z:\home\localho...')
#2 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('Z:\home\localho...', Array, true)
#3 Z:\home\localhost\www\yii\framework\web\CController.php(869): CBaseController->renderFile('Z:\home\localho...', Array, true)
#4 Z:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#5 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): CController->render('index', Array)
#6 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#7 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 Z:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 Z:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#11 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/index')
#12 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii/index.php/my/index
---
2013/06/17 00:27:39 [error] [php] Invalid argument supplied for foreach() (Z:\home\localhost\www\yii\framework\web\helpers\CHtml.php:2053)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require()
#1 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): MyController->renderInternal()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(869): MyController->renderFile()
#3 Z:\home\localhost\www\yii\protected\views\my\index.php(12): MyController->renderPartial()
#4 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require()
#5 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): MyController->renderInternal()
#6 Z:\home\localhost\www\yii\framework\web\CController.php(869): MyController->renderFile()
#7 Z:\home\localhost\www\yii\framework\web\CController.php(782): MyController->renderPartial()
#8 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): MyController->render()
#9 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#10 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#11 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#12 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#13 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#14 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#15 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\views\my\_page\_form.php (2)
in Z:\home\localhost\www\yii\protected\views\my\index.php (12)
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (14)
2013/06/17 00:28:08 [error] [php] Object of class My could not be converted to string (Z:\home\localhost\www\yii\framework\web\helpers\CHtml.php:2071)
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require()
#1 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): MyController->renderInternal()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(869): MyController->renderFile()
#3 Z:\home\localhost\www\yii\protected\views\my\index.php(12): MyController->renderPartial()
#4 Z:\home\localhost\www\yii\framework\web\CBaseController.php(126): require()
#5 Z:\home\localhost\www\yii\framework\web\CBaseController.php(95): MyController->renderInternal()
#6 Z:\home\localhost\www\yii\framework\web\CController.php(869): MyController->renderFile()
#7 Z:\home\localhost\www\yii\framework\web\CController.php(782): MyController->renderPartial()
#8 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): MyController->render()
#9 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#10 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#11 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#12 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#13 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#14 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#15 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\protected\views\my\_page\_form.php (2)
in Z:\home\localhost\www\yii\protected\views\my\index.php (12)
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (14)
2013/06/18 23:38:16 [error] [php] file_get_contents() [<a href='function.file-get-contents'>function.file-get-contents</a>]: Unable to find the wrapper &quot;https&quot; - did you forget to enable it when you configured PHP? (Z:\home\localhost\www\yii\simple_html_dom.php:75)
Stack trace:
#0 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): MyController->parserHTML()
#1 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#5 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#7 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\simple_html_dom.php (75)
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (22)
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (14)
2013/06/18 23:39:28 [error] [php] file_get_contents(http://twitter.com/habrahabr) [<a href='function.file-get-contents'>function.file-get-contents</a>]: failed to open stream: Unable to find the socket transport &quot;ssl&quot; - did you forget to enable it when you configured PHP? (Z:\home\localhost\www\yii\simple_html_dom.php:75)
Stack trace:
#0 Z:\home\localhost\www\yii\protected\controllers\MyController.php(14): MyController->parserHTML()
#1 Z:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): MyController->actionIndex()
#2 Z:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 Z:\home\localhost\www\yii\framework\web\CController.php(286): MyController->runAction()
#4 Z:\home\localhost\www\yii\framework\web\CController.php(265): MyController->runActionWithFilters()
#5 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): MyController->run()
#6 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#7 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 Z:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/my/index
in Z:\home\localhost\www\yii\simple_html_dom.php (75)
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (22)
in Z:\home\localhost\www\yii\protected\controllers\MyController.php (14)
2013/06/22 16:48:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "jquery.js".' in Z:\home\localhost\www\yii\framework\web\CController.php:483
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(270): CController->missingAction('jquery.js')
#1 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('jquery.js')
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/jquery-2.0.1...')
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/my/jquery.js
HTTP_REFERER=http://localhost/yii/index.php/my/index
---
2013/06/22 16:51:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "jquery.js".' in Z:\home\localhost\www\yii\framework\web\CController.php:483
Stack trace:
#0 Z:\home\localhost\www\yii\framework\web\CController.php(270): CController->missingAction('jquery.js')
#1 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('jquery.js')
#2 Z:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('my/jquery.js')
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 Z:\home\localhost\www\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/my/jquery.js
HTTP_REFERER=http://localhost/yii/index.php/my/index
---
2014/06/07 16:24:25 [error] [exception.CException] exception 'CException' with message 'Контроллер SiteController не может найти представление "category/_form".' in F:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('category/_form', NULL, true)
#1 F:\home\localhost\www\yii\protected\controllers\SiteController.php(32): CController->render('category/_form')
#2 F:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#3 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 F:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 F:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 F:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2014/06/07 16:27:04 [error] [exception.CException] exception 'CException' with message 'Контроллер SiteController не может найти представление "category/_form".' in F:\home\localhost\www\yii\framework\web\CController.php:878
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(782): CController->renderPartial('category/_form', NULL, true)
#1 F:\home\localhost\www\yii\protected\controllers\SiteController.php(32): CController->render('category/_form')
#2 F:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#3 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 F:\home\localhost\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 F:\home\localhost\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 F:\home\localhost\www\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/?r=category/index
---
2014/06/07 17:36:43 [error] [php] Object of class Site could not be converted to string (F:\home\localhost\www\yii\protected\views\site\work.php:3)
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 F:\home\localhost\www\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 F:\home\localhost\www\yii\protected\controllers\SiteController.php(30): SiteController->render()
#3 F:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionWork()
#4 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 F:\home\localhost\www\yii\framework\web\CController.php(286): SiteController->runAction()
#6 F:\home\localhost\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 F:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/site/work
in F:\home\localhost\www\yii\protected\views\site\work.php (3)
in F:\home\localhost\www\yii\protected\controllers\SiteController.php (30)
in F:\home\localhost\www\yii\index.php (13)
2014/06/07 19:52:21 [error] [php] fopen(1.obj) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (F:\home\localhost\www\yii\protected\controllers\SiteController.php:29)
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 F:\home\localhost\www\yii\framework\web\CController.php(286): SiteController->runAction()
#2 F:\home\localhost\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 F:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/site/work
in F:\home\localhost\www\yii\protected\controllers\SiteController.php (29)
in F:\home\localhost\www\yii\index.php (13)
2014/06/07 19:52:57 [error] [php] fopen(F:\home\localhost\www\yii\protected\controllers/1.obj) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (F:\home\localhost\www\yii\protected\controllers\SiteController.php:29)
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 F:\home\localhost\www\yii\framework\web\CController.php(286): SiteController->runAction()
#2 F:\home\localhost\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 F:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/site/work
in F:\home\localhost\www\yii\protected\controllers\SiteController.php (29)
in F:\home\localhost\www\yii\index.php (13)
2014/06/07 19:53:17 [error] [php] fopen(F:\home\localhost\www\yii\protected\controllers\1.obj) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (F:\home\localhost\www\yii\protected\controllers\SiteController.php:29)
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 F:\home\localhost\www\yii\framework\web\CController.php(286): SiteController->runAction()
#2 F:\home\localhost\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 F:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/site/work
in F:\home\localhost\www\yii\protected\controllers\SiteController.php (29)
in F:\home\localhost\www\yii\index.php (13)
2014/06/07 19:55:46 [error] [php] fopen(1.obj) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (F:\home\localhost\www\yii\protected\controllers\SiteController.php:29)
Stack trace:
#0 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 F:\home\localhost\www\yii\framework\web\CController.php(286): SiteController->runAction()
#2 F:\home\localhost\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 F:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/site/work
in F:\home\localhost\www\yii\protected\controllers\SiteController.php (29)
in F:\home\localhost\www\yii\index.php (13)
2014/06/08 15:39:21 [error] [php] preg_match_all() [<a href='function.preg-match-all'>function.preg-match-all</a>]: Compilation failed: nothing to repeat at offset 8 (F:\home\localhost\www\yii\protected\components\APPLICATION.php:34)
Stack trace:
#0 F:\home\localhost\www\yii\protected\models\Site.php(9): APPLICATION->__construct()
#1 F:\home\localhost\www\yii\protected\controllers\SiteController.php(28): Site->__construct()
#2 F:\home\localhost\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionWork()
#3 F:\home\localhost\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 F:\home\localhost\www\yii\framework\web\CController.php(286): SiteController->runAction()
#5 F:\home\localhost\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#6 F:\home\localhost\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#7 F:\home\localhost\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 F:\home\localhost\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 F:\home\localhost\www\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/index.php/site/work
in F:\home\localhost\www\yii\protected\components\APPLICATION.php (34)
in F:\home\localhost\www\yii\protected\components\APPLICATION.php (14)
in F:\home\localhost\www\yii\protected\models\Site.php (9)
